ci/jenkins/test-vm.sh

```diff
@@ -139,26 +142,72 @@ function apply_antrea {
     kubectl apply -f ${WORKDIR}/antrea.yml
 }
 
-function clean_vm_agent {
-    echo "====== Cleanup Antrea-Agent Installation on the VM ======"
-    echo "Revert to snapshot VMAgent"
-    govc snapshot.revert -k=true -vm=${LIN_HOSTNAME} VMAgent
-    govc snapshot.revert -k=true -vm=${WIN_HOSTNAME} VMAgent
-    echo "Get IP addresses for VMs"
-    UBUNTU_VM_IP=$(govc vm.ip -k=true -wait=1m ${LIN_HOSTNAME})
-    for i in `seq 10`; do
-        WINDOWS_VM_IP=$(govc vm.ip -k=true -wait=2m ${WIN_HOSTNAME})
-        if [[ WINDOWS_VM_IP == "" ]]; then
-            echo "Failed to retrieve IP for Windows VM ${WIN_HOSTNAME}, retry ${i}"
-            continue
-        fi
+function export_govc_env_var {
+    # This should be coming from jenkins configuration
+    export GOVC_URL=$GOVC_URL
+    export GOVC_USERNAME=$GOVC_USERNAME
+    export GOVC_PASSWORD=$GOVC_PASSWORD
+    export GOVC_INSECURE=1
+    export GOVC_DATACENTER=$GOVC_DATACENTER
+    export GOVC_DATASTORE=$GOVC_DATASTORE
+}
+
+function clean_up_one_ns {
+    ns=$1
+    kubectl get pod -n "${ns}" --no-headers=true | awk '{print $1}' | while read pod_name; do
```

Comment:
You can use this command to delete all Pods in one namespace,

```
kubectl delete pods -n${ns} --all
```

I think the Pods should be removed automatically when you delete the namespace, so maybe you don't need this command?

Reply:
Done.